### PR TITLE
New package: ncpamixer-git-20180705_1

### PR DESCRIPTION
**Pull request by benalb**

### CODE REVIEW

File: srcpkgs/ncpamixer-git/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://github.com/fulhax/ncpamixer"
+provides="ncpamixer-${version}_${revision}"
+replaces="ncpamixer>=0"
+
+do_fetch() {
+	local url="git://github.com/fulhax/ncpamixer.git"
+	msg_normal "Fetching source from $url ...\n"
+	git clone ${url} ${pkgname}-${version}
+}
+	do_build() {
```
Comment (abenson) [inline srcpkgs/ncpamixer-git/template]: don't indent these

File: srcpkgs/ncpamixer-git/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	make
+}
+	do_install() {
```
Comment (abenson) [inline srcpkgs/ncpamixer-git/template]: same here

File: srcpkgs/ncpamixer-git/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+makedepends="pulseaudio-devel ncurses-devel"
+short_desc="An ncurses mixer for PulseAudio"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="MIT"
+homepage="https://github.com/fulhax/ncpamixer"
+provides="ncpamixer-${version}_${revision}"
+replaces="ncpamixer>=0"
+
+do_fetch() {
+	local url="git://github.com/fulhax/ncpamixer.git"
```
Comment (abenson) [inline srcpkgs/ncpamixer-git/template]: Why not just use the tagged release?

File: srcpkgs/ncpamixer-git/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+version=20180705
+revision=1
+hostmakedepends="git cmake"
+makedepends="pulseaudio-devel ncurses-devel"
+short_desc="An ncurses mixer for PulseAudio"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
+license="MIT"
+homepage="https://github.com/fulhax/ncpamixer"
+provides="ncpamixer-${version}_${revision}"
+replaces="ncpamixer>=0"
```
Comment (abenson) [inline srcpkgs/ncpamixer-git/template]: I don't see an existing ncpamixer

File: srcpkgs/ncpamixer-git/template (hunk 1 above)
Comment (cr6git) [reply srcpkgs/ncpamixer-git/template]: @benalb not possible to just use `build_style=gnu-makefile` ?

Comment (abenson) [reply srcpkgs/ncpamixer-git/template]: It should be `build_style=cmake` and `build_wrksrc=src`

Comment (abenson) [reply srcpkgs/ncpamixer-git/template]: This was the template I did: https://github.com/abenson/void-packages/blob/39ae19e6f360690275b946664dfc5adeed3746e0/srcpkgs/ncpamixer/template

Comment (benalb) [reply srcpkgs/ncpamixer-git/template]: I took the ttyload-git template as base to make the ncpamixer one. 
Now, as this is my first PR, get the template suggested by abenson and update the PR or ...?

Comment (abenson) [reply srcpkgs/ncpamixer-git/template]: Feel free to use my template. 

The `ttyload-git` template is really old. The current methods that seem to be preferred for Void is to preferably use tagged versions, or in rarer cases, use a git commit hash if the usptream doesn't tag properly. That doesn't seem to be an issue with this project, so it's recommended that the tag be used.

Comment (benalb) [reply srcpkgs/ncpamixer-git/template]: Now, should I rename ncpamixer-git to ncpamixer, and then edit the template?

Comment (abenson) [reply srcpkgs/ncpamixer-git/template]: Yes.

**Comment by cr6git** [comment]
@benalb no, one commit is enough.

**Comment by abenson** [comment]
@benalb If you're not familiar with squashing commits, https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by benalb** [comment]
Before I break something more.
In my local repo, 

git rebase -i HEAD~3

and mark all three with squash

and then git push  -force to my github repo , right?

**Comment by Anachron** [comment]
Yep

**Comment by abenson** [comment]
No, I think you only want to squash the second two into the first, and then make sure the first reads correctly.

**Comment by Anachron** [comment]
But he has 3 commits? Every PR should only have one commit.

**Comment by cr6git** [comment]
Can someone please take a look at https://github.com/fulhax/ncpamixer/issues/15 ?

**Comment by benalb** [comment]
Can not replicate, with:
kernel 4.16.14_1
ncurses-6.1_1
pulseaudio-12.0_1
firefox-61.0_2

Playing a movie with mpv and simultaneously a youtube video in firefox, can scroll/seek in both without crash.

**Comment by cr6git** [comment]
@benalb thanks! :+1:

**Pull request merged**